src/components/Inputs/input.module.scss

```diff
@@ -489,6 +451,12 @@
             }
         }
 
+        &.inline {
+            display: flex;
+            flex-direction: column;
+            flex-grow: 1;
+        }
```

Comment:
I'm guessing at some point someone is going to want to stack inline elements on top of each other and have their inputs line up. Give that some thought but try not to lose sleep over it. I'm assuming for now the use case here is above tables, etc. places where we want the vertical spacing minimized